examples/bratko_kopec/bratko_kopec.py

```diff
@@ -63,7 +65,6 @@ def test_epd_with_fractional_scores(engine, epd, VariantBoard, movetime):
     score = 0.0
 
     print("%s:" % epd_string, end=" ")
-    sys.stdout.flush()
```

Comment:
I think I had originally added the `flush` call because I didn't know how to run python in the background with stdout unbuffered (`python -u`).  This code seems unnecessary when running in the foreground, since stdout is unbuffered.